README.md

```diff
@@ -11,8 +11,8 @@
     <a href="https://redscientist.com/rtc"><img src="https://img.shields.io/badge/download-RTC-red.svg" alt="Download badge" /></a>
     <!-- Discord -->
     <a href="https://corrupt.wiki/corruptors/rtc/expert#rtc-dev-discord"><img src="https://img.shields.io/discord/279664862836031488.svg" alt="Chat badge" /></a>
-    <!-- Trello -->
-    <a href="https://trello.com/b/9QYo50OC/rtcv"><img src="https://img.shields.io/badge/planning-Trello-blue.svg" alt="Trello badge" /></a>
+    <!-- CodeCov -->
+    <a href="https://codecov.io/gh/redscientistlabs/RTCV"><img alt="Codecov" src="https://img.shields.io/codecov/c/github/redscientistlabs/RTCV"></a>
```

Comment:
CodeCov would need to be enabled for the repository for the coverage reporting to work

Reply:
I'm fine with bringing CodeCov in. It's going to be a whopping F with a 0.1% coverage but maybe that badge will be a motivator to get some actual coverage.

Reply:
I've enabled CodeCov on the repo. Just gotta pull in the action and it should work.

Reply:
I've brought it in. Once the run succeeds I'll merge this and we can get started on writing some tests 🎈 😢 🎈

Reply:
Perfect, thank you!